tests/e2e/e2e_setup_test.go

```diff
@@ -547,12 +600,42 @@ func (s *IntegrationTestSuite) upgrade() {
 		}
 	}
 
-	// remove all containers so we can upgrade them to the new version
 	for _, chainConfig := range s.chainConfigs {
 		s.upgradeContainers(chainConfig, chainConfig.propHeight)
 	}
 }
 
+func (s *IntegrationTestSuite) upgradeFork() {
```

Comment:
Please add to README documenting how the fork logic works, including changing `initial_height` in genesis